### ISSUE TITLE
[FR] Allow configuration of load / unload speeds in M603 to affect M600 / M701 / M702

### ISSUE THREAD

**Issue by CRCinAU** [MarlinFirmware/Marlin#19548]
### Description

Currently, we are able to set the unload and load distances for M600/701/702 via the M603 command. The only thing missing on this is the speed at which the unload / load occurs at.

It makes sense to put this within the M603 command as parameters, as the existing load / unload distance commands are already present.

**Comment by inigoml** [comment]
Agree. In my case, LOAD SPEED is too fast and causes problems with extruder. So, I've to do it manually.
